doc/manual/modules/performance/ref_available-tuned-plug-ins.adoc

```diff
@@ -139,38 +139,92 @@ The powersave level can be specified by using the [option]`radeon_powersave` opt
 For details, see link:http://www.x.org/wiki/RadeonFeature#KMS_Power_Management_Options[www.x.org]. Note that this plug-in is experimental and the option might change in future releases. 
 
 `bootloader`::
-Adds options to the kernel command line. This plug-in supports only the GRUB 2 boot loader.
+Adds options to the kernel command line. This plug-in supports the GRUB 2 boot loader and the Boot Loader Specification (BLS).
 +
 Customized non-standard location of the GRUB 2 configuration file can be specified by the [option]`grub2_cfg_file` option.
 +
 The kernel options are added to the current GRUB configuration and its templates. The system needs to be rebooted for the kernel options to take effect.
```

Comment:
*NOTE* `tuned` will *not* remove or replace bootloader cmdline parameters added via other methods like `grubby`.  `tuned` will only be able manage bootloader cmdline parameters added via `tuned`.  Please refer to your platform bootloader documentation about how to identify and manage bootloader cmdline parameters set outside of `tuned`.

Reply:
@richm thanks added.